src/shell/hooks/useFilePath.js

```diff
@@ -14,15 +18,31 @@ export function useFilePath(ZUID = "") {
   // There is no place where this method is called by a content item zuid, if there was we would need to reference all items.
   switch (Number(ZUID.split("-")[0])) {
     case zuid.prefix.SITE_CONTENT_SET:
-      let findfile = files.find(file => file.contentModelZUID === ZUID);
+      let findfile = files.find((file) => file.contentModelZUID === ZUID);
       if (findfile) {
         return `/code/file/views/${findfile.ZUID}`;
       } else {
         return "/code";
       }
 
+    // Searching on Content Item Zuid 7-
+    case zuid.prefix.SITE_CONTENT_ITEM:
+      let findItemZUID = Object.keys(content).find((file) => file === ZUID);
```

Comment:
If content items is an object keyed by ZUID then we should be able to do a direct item look up. e.g. 
```
const item = content[ZUID]
```

You will need to confirm the item is present in full as we pre-populate them when fetching publish records. I.e. you'll need to confirm the `item.meta.contentModelZUID` is present before finding it's corresponding file match.